### PR TITLE
Mobile View Responsive

### LINKED ISSUE #19: Make content responsive for (Mobile view)
## Description: 
Make `Home Page` content responsive for (Mobile view).

### PR DESCRIPTION
**Pull request by m-code12**
# Description
Include a summary of the change and relevant motivation/context. List any dependencies that are required for this change.

Fixes #19 

# Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation

# How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.

# Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by m-code12** [comment]
@aashimgarg Please review my pr.

File: index.html [hunk 1]
```diff
@@ -19,6 +19,22 @@
...
+}
+  
+  #para3 {
+  font-size: 35px;
+  font-weight: 1400;
+}
+
+
+
+</style>
```
Comment (aashimgarg) [inline index.html:38]: hey @m-code12 can you please move these CSS content to the respective CSS file

File: index.html [hunk 2]
```diff
@@ -76,40 +92,72 @@ <h3 class="quotes"><em>“When you don't create things, you become defined by yo
...
 
 
 
-  </div>
 
-  <div id="months">
+
+
+ 
+
```
Comment (aashimgarg) [inline index.html:159]: Please try to minimize the unused spaces.

**Comment by m-code12** [comment]
I have made the changes . Please review and merge @aashimgarg

**Comment by m-code12** [comment]
@aashimgarg Please merge this pr . I have made the requested changes .

**Pull request merged**